gravity.c

```diff
@@ -49,16 +50,20 @@ static bool gravityDB_open(void)
 	return true;
 }
 
+// Close gravity database handle
 static void gravityDB_close(void)
 {
-	// Close database handle
 	sqlite3_close(gravitydb);
 }
 
+// Prepare a SQLite3 statement which can be used by
+// gravityDB_getDomain() to get blocking domains from
+// a table which is specified when calling this function
 bool gravityDB_getTable(const unsigned char list)
 {
 	// Open gravity database, fail is not possible
```

Comment:
Instead of saying "not possible", perhaps say "stop execution if open fails" or nothing at all?